src/protagonist/API/Features/Image/Ingest/DeliveryChannelProcessor.cs

```diff
@@ -73,20 +72,20 @@ private bool DeliveryChannelsRequireReprocessing(Asset originalAsset, DeliveryCh
         return false;
     }
 
-    private async Task<bool> SetImageDeliveryChannels(Asset asset, DeliveryChannelsBeforeProcessing[] deliveryChannelsBeforeProcessing, bool isUpdate)
+    private async Task<bool> SetImageDeliveryChannels(Asset asset, DeliveryChannelsBeforeProcessing[]? deliveryChannelsBeforeProcessing, bool isUpdate)
     {
         var assetId = asset.Id;
+        var isDefaultChannel = deliveryChannelsBeforeProcessing?.Any(d => d.Channel == AssetDeliveryChannels.Default) ??
+                               false;
         
-        if (!isUpdate)
+        // if it's a new asset, or has "default" specified, figure out the delivery channels based on media type
+        if (!isUpdate || isDefaultChannel)
         {
-            logger.LogTrace("Asset {AssetId} is new, resetting ImageDeliveryChannels", assetId);
+            logger.LogTrace("Setting delivery channels for asset {AssetId}", assetId);
             asset.ImageDeliveryChannels = new List<ImageDeliveryChannel>();
-            
-            // Only valid for creation - set image delivery channels to default values for media type
-            if (deliveryChannelsBeforeProcessing.IsNullOrEmpty())
+
+            if (deliveryChannelsBeforeProcessing.IsNullOrEmpty() || isDefaultChannel)
             {
-                logger.LogDebug("Asset {AssetId} is new, no deliveryChannels specified. Assigning defaults for mediaType",

```

Comment:
Worth keeping log saying we're resetting delivery-channels to aid in debugging scenarios?